
<!DOCTYPE HTML>

<html lang="en" class="_theme-default">

<head>
  <meta charset="utf-8">
  <title>Std&#58;&#58;overflow_error - C++ - W3cubDocs</title>
  
  <meta name="description" content="Defines a type of object to be thrown as exception. It can be used to report arithmetic overflow errors (that is, situations where a result of a &hellip;">
  <meta name="keywords" content="std, overflow, error, c++, cpp">
  <meta name="HandheldFriendly" content="True">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="mobile-web-app-capable" content="yes">
  
  <link rel="canonical" href="https://docs.w3cub.com/cpp/error/overflow_error.html">
  <link href="/favicon.png" rel="icon">
  <link rel="stylesheet" type="text/css" href="/assets/application-9c7b0d969c0ce639a6c01c2badcdd81667c7c66206aa3d0fccc5cb0585fb1b912dab72ea882101d18010680927514b4b77b107282db8f756fbf63301fe9c88b0.css">
  <script type="text/javascript" src="/assets/application-79c555f6b25481fffac2cac30a7f3e54e608ca09e9e8e42bb1790095ba6d0fcace47d6bc624ddce952c70370892f2d46864f89e6943d4f7f7ff16c8a3231a91a.js"></script>
  <script src="/json/cpp.js"></script>
  
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-R3WC07G3GB"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'G-R3WC07G3GB');
</script>
<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-2572770204602497"
     crossorigin="anonymous"></script>
<script async custom-element="amp-auto-ads"
  src="https://cdn.ampproject.org/v0/amp-auto-ads-0.1.js">
</script>


</head>

<body class="docs">
	<amp-auto-ads type="adsense"
              data-ad-client="ca-pub-2572770204602497">
	</amp-auto-ads>
	<div class="_app">
	<header class="_header">

  <a class="_home-link" href="/" ></a>
  <a class="_menu-link"></a>
  <h1 class="_logo">
    <a href="/" class="_nav-link" title="API Documentation Browser">W3cubDocs</a>
  </h1>
  
  <span class="_logo-sub-nav">/</span><span class="_logo-sub-nav"><a href="/cpp/" class="_nav-link" title="" style="margin-left:0;">C++</a></span>
  
  <nav class="_nav">
    <a href="https://tools.w3cub.com/?_sp=docs" target="_blank" class="_nav-link ">W3cubTools</a>
    <a href="/cheatsheets/" class="_nav-link ">Cheatsheets</a>
    <a href="/about/" class="_nav-link ">About</a>
  </nav>
</header>
	<section class="_sidebar">
		
		<form class="_search">
		  <input type="search" class="_search-input" placeholder="Search&hellip;" autocomplete="off" autocapitalize="off" autocorrect="off" spellcheck="false" maxlength="20">
		  <a class="_search-clear"></a>
		  <div class="_search-tag"></div>
		</form>
		
		<div class="_list-wrap">
			<div class="_list">
			
			</div>
		</div>
	</section>
	<section class="_container ">
		<div class="_content">
			<ins class="adsbygoogle"
     style="display:block"
     data-ad-client="ca-pub-2572770204602497"
     data-ad-slot="6861657091"
     data-ad-format="auto"
     data-full-width-responsive="true"></ins>
<script>
     (adsbygoogle = window.adsbygoogle || []).push({});
</script>
			<div class="_page _cppref">
				
				
<h1 id="firstHeading" class="firstHeading">std::overflow_error</h1>            <table class="t-dcl-begin"> <tr class="t-dsc-header"> <th> Defined in header <code><a href="../header/stdexcept" title="cpp/header/stdexcept">&lt;stdexcept&gt;</a></code> </th> <th> </th> <th> </th> </tr> <tr class="t-dcl"> <td class="t-dcl-nopad"> <pre data-language="cpp">class overflow_error;</pre>
</td> <td class="t-dcl-nopad"> </td> <td class="t-dcl-nopad"> </td> </tr>  </table> <p>Defines a type of object to be thrown as exception. It can be used to report arithmetic overflow errors (that is, situations where a result of a computation is too large for the destination type).</p>
<table class="t-rev-begin"> <tr class="t-rev t-until-cxx11">
<td> <p>The only standard library component that throws this exception is <code><a href="../utility/bitset/to_ulong" title="cpp/utility/bitset/to ulong">std::bitset::to_ulong</a></code>.</p>
</td> <td><span class="t-mark-rev t-until-cxx11">(until C++11)</span></td>
</tr> <tr class="t-rev t-since-cxx11">
<td> <p>The only standard library components that throw this exception are <code><a href="../utility/bitset/to_ulong" title="cpp/utility/bitset/to ulong">std::bitset::to_ulong</a></code> and <code><a href="../utility/bitset/to_ullong" title="cpp/utility/bitset/to ullong">std::bitset::to_ullong</a></code>.</p>
</td> <td><span class="t-mark-rev t-since-cxx11">(since C++11)</span></td>
</tr> </table> <p>The mathematical functions of the standard library components do not throw this exception (mathematical functions report overflow errors as specified in <code><a href="../numeric/math/math_errhandling" title="cpp/numeric/math/math errhandling">math_errhandling</a></code>). Third-party libraries, however, use this. For example, <a rel="nofollow" class="external text" href="https://www.boost.org/doc/libs/release/libs/math/doc/html/math_toolkit/error_handling.html">boost.math</a> throws <code>std::overflow_error</code> if <code>boost::math::policies::throw_on_error</code> is enabled (the default setting).</p>
<div class="t-inheritance-diagram"> <div class="center"><div class="noresize">
<map name="ImageMap_1_542650028"><area href="/w/cpp/error/exception" shape="rect" coords="5,4,107,36" alt="cpp/error/exception" title="cpp/error/exception">
<area href="/w/cpp/error/runtime_error" shape="rect" coords="135,4,238,36" alt="cpp/error/runtime error" title="cpp/error/runtime error"></map><img alt="std-overflow error-inheritance.svg" src="data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+CjwhRE9DVFlQRSBzdmcgUFVCTElDICItLy9XM0MvL0RURCBTVkcgMS4xLy9FTiIgImh0dHA6Ly93d3cudzMub3JnL0dyYXBoaWNzL1NWRy8xLjEvRFREL3N2ZzExLmR0ZCI+CjwhLS0gR2VuZXJhdGVkIGJ5IGdyYXBodml6IHZlcnNpb24gMi4yNi4zICgyMDEwMDEyNi4xNjAwKQogLS0+CjwhLS0gVGl0bGU6IGluaGVyaXRhbmNlIFBhZ2VzOiAxIC0tPgo8c3ZnIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHdpZHRoPSIzNzdweCIgaGVpZ2h0PSI0MHB4IiB2aWV3Qm94PSIwLjAwIDAuMDAgNDE4LjAwIDQ0LjAwIj4KPGcgaWQ9ImdyYXBoMSIgY2xhc3M9ImdyYXBoIiB0cmFuc2Zvcm09InJvdGF0ZSgwKSB0cmFuc2xhdGUoNCA0MCkiPgo8dGl0bGU+aW5oZXJpdGFuY2U8L3RpdGxlPgo8IS0tIGUxIC0tPgo8ZyBpZD0ibm9kZTEiIGNsYXNzPSJub2RlIj48dGl0bGU+ZTE8L3RpdGxlPgo8YT4KPHBvbHlnb24gZmlsbD0id2hpdGUiIHN0cm9rZT0iYmxhY2siIHBvaW50cz0iMTE1LC0zNiAxLC0zNiAxLC0xLjc3NjM2ZS0xNCAxMTUsLTMuNTUyNzFlLTE1IDExNSwtMzYiLz4KPHRleHQgdGV4dC1hbmNob3I9Im1pZGRsZSIgeD0iNTgiIHk9Ii0xNC40IiBmb250LWZhbWlseT0iRGVqYVZ1IFNhbnMsIHZlcmRhbmEsIHNhbnMtc2VyaWYiIGZvbnQtc2l6ZT0iMTQuMDAiPmV4Y2VwdGlvbjwvdGV4dD4KPC9hPgo8L2c+CjwhLS0gZTIgLS0+CjxnIGlkPSJub2RlMiIgY2xhc3M9Im5vZGUiPjx0aXRsZT5lMjwvdGl0bGU+CjxhPgo8cG9seWdvbiBmaWxsPSJ3aGl0ZSIgc3Ryb2tlPSJibGFjayIgcG9pbnRzPSIyNjAsLTM2IDE0NiwtMzYgMTQ2LC0xLjc3NjM2ZS0xNCAyNjAsLTMuNTUyNzFlLTE1IDI2MCwtMzYiLz4KPHRleHQgdGV4dC1hbmNob3I9Im1pZGRsZSIgeD0iMjAzIiB5PSItMTQuNCIgZm9udC1mYW1pbHk9IkRlamFWdSBTYW5zLCB2ZXJkYW5hLCBzYW5zLXNlcmlmIiBmb250LXNpemU9IjE0LjAwIj5ydW50aW1lX2Vycm9yPC90ZXh0Pgo8L2E+CjwvZz4KPCEtLSBlMiYjNDU7Jmd0O2UxIC0tPgo8ZyBpZD0iZWRnZTIiIGNsYXNzPSJlZGdlIj48dGl0bGU+ZTItJmd0O2UxPC90aXRsZT4KPHBhdGggZmlsbD0ibm9uZSIgc3Ryb2tlPSJibGFjayIgZD0iTTE0NS4yNzYsLTE4QzE0MC41MzIsLTE4IDEzNS43LC0xOCAxMzAuODY3LC0xOCIvPgo8cG9seWdvbiBmaWxsPSJub25lIiBzdHJva2U9ImJsYWNrIiBwb2ludHM9IjEzMC41MjYsLTEyLjc1MDEgMTE1LjUyNiwtMTggMTMwLjUyNiwtMjMuMjUwMSAxMzAuNTI2LC0xMi43NTAxIi8+CjwvZz4KPCEtLSBlMyAtLT4KPGcgaWQ9Im5vZGUzIiBjbGFzcz0ibm9kZSI+PHRpdGxlPmUzPC90aXRsZT4KPGE+Cjxwb2x5Z29uIGZpbGw9ImxpZ2h0Z3JleSIgc3Ryb2tlPSJibGFjayIgcG9pbnRzPSI0MDksLTM2IDI5MSwtMzYgMjkxLC0xLjc3NjM2ZS0xNCA0MDksLTMuNTUyNzFlLTE1IDQwOSwtMzYiLz4KPHRleHQgdGV4dC1hbmNob3I9Im1pZGRsZSIgeD0iMzUwIiB5PSItMTQuNCIgZm9udC1mYW1pbHk9IkRlamFWdSBTYW5zLCB2ZXJkYW5hLCBzYW5zLXNlcmlmIiBmb250LXNpemU9IjE0LjAwIj5vdmVyZmxvd19lcnJvcjwvdGV4dD4KPC9hPgo8L2c+CjwhLS0gZTMmIzQ1OyZndDtlMiAtLT4KPGcgaWQ9ImVkZ2U0IiBjbGFzcz0iZWRnZSI+PHRpdGxlPmUzLSZndDtlMjwvdGl0bGU+CjxwYXRoIGZpbGw9Im5vbmUiIHN0cm9rZT0iYmxhY2siIGQ9Ik0yOTAuMjA5LC0xOEMyODUuNTEyLC0xOCAyODAuNzM4LC0xOCAyNzUuOTY2LC0xOCIvPgo8cG9seWdvbiBmaWxsPSJub25lIiBzdHJva2U9ImJsYWNrIiBwb2ludHM9IjI3NS44MzUsLTEyLjc1MDEgMjYwLjgzNSwtMTggMjc1LjgzNSwtMjMuMjUwMSAyNzUuODM1LC0xMi43NTAxIi8+CjwvZz4KPC9nPgo8L3N2Zz4K" width="377" height="40" usemap="#ImageMap_1_542650028">
</div></div> <div style="text-align:center;"> <p>Inheritance diagram.</p>
</div> </div>  <h3 id="Member_functions"> Member functions</h3> <table class="t-dsc-begin"> <tr class="t-dsc"> <td> <div><span class="t-lines"><span>(constructor)</span></span></div> </td> <td> constructs a new <code>overflow_error</code> object with the given message <br> <span class="t-mark">(public member function)</span> </td>
</tr> <tr class="t-dsc"> <td> <div><span class="t-lines"><span>operator=</span></span></div> </td> <td> replaces the <code>overflow_error</code> object <br> <span class="t-mark">(public member function)</span> </td>
</tr> </table> <div class="t-member"> <h2 id="std::overflow_error::overflow_error"> std::overflow_error::overflow_error</h2> <table class="t-dcl-begin"> <tr class="t-dcl"> <td> <pre data-language="cpp">overflow_error( const std::string&amp; what_arg );</pre>
</td> <td> (1) </td> <td class="t-dcl-nopad"> </td> </tr> <tr class="t-dcl"> <td> <pre data-language="cpp">overflow_error( const char* what_arg );</pre>
</td> <td> (2) </td> <td class="t-dcl-nopad"> </td> </tr>  <tr class="t-dcl-rev-aux"> <td></td> <td rowspan="3">(3)</td> <td></td> </tr> <tr class="t-dcl t-until-cxx11"> <td> <pre data-language="cpp">overflow_error( const overflow_error&amp; other );</pre>
</td>  <td> <span class="t-mark-rev t-until-cxx11">(until C++11)</span> </td> </tr> <tr class="t-dcl t-since-cxx11"> <td> <pre data-language="cpp">overflow_error( const overflow_error&amp; other ) noexcept;</pre>
</td>  <td> <span class="t-mark-rev t-since-cxx11">(since C++11)</span> </td> </tr>   </table> <div class="t-li1">
<span class="t-li">1)</span> Constructs the exception object with <code>what_arg</code> as explanatory string. After construction, <code><a href="http://en.cppreference.com/w/cpp/string/byte/strcmp"><span class="kw1149">std::<span class="me2">strcmp</span></span></a><span class="br0">(</span>what<span class="br0">(</span><span class="br0">)</span>, what_arg.<span class="me1">c_str</span><span class="br0">(</span><span class="br0">)</span><span class="br0">)</span> <span class="sy1">==</span> <span class="nu0">0</span></code>.</div> <div class="t-li1">
<span class="t-li">2)</span> Constructs the exception object with <code>what_arg</code> as explanatory string. After construction, <code><a href="http://en.cppreference.com/w/cpp/string/byte/strcmp"><span class="kw1149">std::<span class="me2">strcmp</span></span></a><span class="br0">(</span>what<span class="br0">(</span><span class="br0">)</span>, what_arg<span class="br0">)</span> <span class="sy1">==</span> <span class="nu0">0</span></code>.</div> <div class="t-li1">
<span class="t-li">3)</span> Copy constructor. If <code>*this</code> and <code>other</code> both have dynamic type <code>std::overflow_error</code> then <code><a href="http://en.cppreference.com/w/cpp/string/byte/strcmp"><span class="kw1149">std::<span class="me2">strcmp</span></span></a><span class="br0">(</span>what<span class="br0">(</span><span class="br0">)</span>, other.<span class="me1">what</span><span class="br0">(</span><span class="br0">)</span><span class="br0">)</span> <span class="sy1">==</span> <span class="nu0">0</span></code>. <span class="t-rev-inl t-until-cxx11"><span>No exception can be thrown from the copy constructor.</span> <span><span class="t-mark-rev t-until-cxx11">(until C++11)</span></span></span>
</div> <h3 id="Parameters"> Parameters</h3> <table class="t-par-begin"> <tr class="t-par"> <td> what_arg </td> <td> - </td> <td> explanatory string </td>
</tr> <tr class="t-par"> <td> other </td> <td> - </td> <td> another exception object to copy </td>
</tr>
</table> <h3 id="Exceptions"> Exceptions</h3> <div class="t-li1">
<span class="t-li">1-2)</span> May throw <code><a href="../memory/new/bad_alloc" title="cpp/memory/new/bad alloc">std::bad_alloc</a></code>
</div> <h3 id="Notes"> Notes</h3> <p>Because copying <code>std::overflow_error</code> is not permitted to throw exceptions, this message is typically stored internally as a separately-allocated reference-counted string. This is also why there is no constructor taking <code>std::string&amp;&amp;</code>: it would have to copy the content anyway.</p>
<p>Before the resolution of <a rel="nofollow" class="external text" href="https://cplusplus.github.io/LWG/issue254">LWG issue 254</a>, the non-copy constructor can only accept <code><a href="../string/basic_string" title="cpp/string/basic string">std::string</a></code>. It makes dynamic allocation mandatory in order to construct a <code><a href="../string/basic_string" title="cpp/string/basic string">std::string</a></code> object.</p>
<p>After the resolution of <a rel="nofollow" class="external text" href="https://cplusplus.github.io/LWG/issue471">LWG issue 471</a>, a derived standard exception class must have a publicly accessible copy constructor. It can be implicitly defined as long as the explanatory strings obtained by <code>what()</code> are the same for the original object and the copied object.</p>
</div> <div class="t-member"> <h2 id="std::overflow_error::operator.3D"> std::overflow_error::operator=</h2> <table class="t-dcl-begin">   <tr class="t-dcl t-until-cxx11"> <td> <pre data-language="cpp">overflow_error&amp; operator=( const overflow_error&amp; other );</pre>
</td> <td class="t-dcl-nopad"> </td> <td> <span class="t-mark-rev t-until-cxx11">(until C++11)</span> </td> </tr> <tr class="t-dcl t-since-cxx11"> <td> <pre data-language="cpp">overflow_error&amp; operator=( const overflow_error&amp; other ) noexcept;</pre>
</td> <td class="t-dcl-nopad"> </td> <td> <span class="t-mark-rev t-since-cxx11">(since C++11)</span> </td> </tr>   </table> <p>Assigns the contents with those of <code>other</code>. If <code>*this</code> and <code>other</code> both have dynamic type <code>std::overflow_error</code> then <code><a href="http://en.cppreference.com/w/cpp/string/byte/strcmp"><span class="kw1149">std::<span class="me2">strcmp</span></span></a><span class="br0">(</span>what<span class="br0">(</span><span class="br0">)</span>, other.<span class="me1">what</span><span class="br0">(</span><span class="br0">)</span><span class="br0">)</span> <span class="sy1">==</span> <span class="nu0">0</span></code> after assignment. <span class="t-rev-inl t-until-cxx11"><span>No exception can be thrown from the copy assignment operator.</span> <span><span class="t-mark-rev t-until-cxx11">(until C++11)</span></span></span>.</p>
<h3 id="Parameters_2"> Parameters</h3> <table class="t-par-begin"> <tr class="t-par"> <td> other </td> <td> - </td> <td> another exception object to assign with </td>
</tr>
</table> <h3 id="Return_value"> Return value</h3> <p><code>*this</code>.</p>
<h3 id="Notes_2"> Notes</h3> <p>After the resolution of <a rel="nofollow" class="external text" href="https://cplusplus.github.io/LWG/issue471">LWG issue 471</a>, a derived standard exception class must have a publicly accessible copy assignment operator. It can be implicitly defined as long as the explanatory strings obtained by <code>what()</code> are the same for the original object and the copied object.</p>
</div> <div class="t-inherited"> <h2 id="Inherited_from_std::runtime_error"> Inherited from <a href="runtime_error" title="cpp/error/runtime error"> std::runtime_error</a> </h2> </div> <div class="t-inherited"> <h2 id="Inherited_from_std::exception"> Inherited from <a href="exception" title="cpp/error/exception"> std::exception</a> </h2> <h3 id="Member_functions_2"> Member functions</h3> <table class="t-dsc-begin"> <tr class="t-dsc"> <td> <div><a href="exception/~exception" title="cpp/error/exception/~exception"> <span class="t-lines"><span>(destructor)</span></span></a></div>
<div><span class="t-lines"><span><span class="t-cmark">[virtual]</span></span></span></div> </td> <td> destroys the exception object <br> <span class="t-mark">(virtual public member function of <code>std::exception</code>)</span>  </td>
</tr> <tr class="t-dsc"> <td> <div><a href="exception/what" title="cpp/error/exception/what"> <span class="t-lines"><span>what</span></span></a></div>
<div><span class="t-lines"><span><span class="t-cmark">[virtual]</span></span></span></div> </td> <td> returns an explanatory string <br> <span class="t-mark">(virtual public member function of <code>std::exception</code>)</span>  </td>
</tr> </table> </div> <h3 id="Example"> Example</h3> <div class="t-example"> <div class="cpp source-cpp"><pre data-language="cpp">#include &lt;iostream&gt;
#include &lt;limits&gt;
#include &lt;stdexcept&gt;
#include &lt;utility&gt;
 
template &lt;typename T, int N&gt;
    requires (N &gt; 0) /*...*/
class Stack
{
    int top_ { -1 };
    T data_[N];
 
public:
    [[nodiscard]] bool empty() const { return top_ == -1; }
 
    void push(T x)
    {
        if (top_ == N - 1)
            throw std::overflow_error("Stack overflow!");
        data_[++top_] = std::move(x);
    }
 
    void pop()
    {
        if (empty())
            throw std::overflow_error("Stack underflow!");
        top_--;
    }
 
    T const&amp; top() const
    {
        if (empty())
            throw std::overflow_error("Stack is empty!");
        return data_[top_];
    }
};
 
int main()
{
    Stack&lt;int, 4&gt; st;
 
    try
    {
        [[maybe_unused]] auto x = st.top();
    }
    catch (std::overflow_error const&amp; ex)
    {
        std::cout &lt;&lt; "1) Exception: " &lt;&lt; ex.what() &lt;&lt; '\n';
    }
 
    st.push(1337);
    while (not st.empty())
            st.pop();
 
    try
    {
        st.pop();
    }
    catch (std::overflow_error const&amp; ex)
    {
        std::cout &lt;&lt; "2) Exception: " &lt;&lt; ex.what() &lt;&lt; '\n';
    }
 
    try
    {
        for (int i {}; i != 13; ++i)
            st.push(i);
    }
    catch (std::overflow_error const&amp; ex)
    {
        std::cout &lt;&lt; "3) Exception: " &lt;&lt; ex.what() &lt;&lt; '\n';
    }
}</pre></div> <p>Output:</p>
<div class="text source-text"><pre data-language="c">1) Exception: Stack is empty!
2) Exception: Stack underflow!
3) Exception: Stack overflow!</pre></div> </div> <h3 id="Defect_reports"> Defect reports</h3> <p>The following behavior-changing defect reports were applied retroactively to previously published C++ standards.</p>
<table class="dsctable"> <tr> <th> DR </th> <th> Applied to </th> <th> Behavior as published </th> <th> Correct behavior </th>
</tr> <tr> <td> <a rel="nofollow" class="external text" href="https://cplusplus.github.io/LWG/issue254">LWG 254</a> </td> <td> C++98 </td> <td> the constructor accepting <code>const char*</code> was missing </td> <td> added </td>
</tr> <tr> <td> <a rel="nofollow" class="external text" href="https://cplusplus.github.io/LWG/issue471">LWG 471</a> </td> <td> C++98 </td> <td> the explanatory strings of <code>std::overflow_error</code>'s<br>copies were implementation-defined </td> <td> they are the same as that of the<br>original <code>std::overflow_error</code> object </td>
</tr>
</table>           <div class="_attribution">
  <p class="_attribution-p">
    © cppreference.com<br>Licensed under the Creative Commons Attribution-ShareAlike Unported License v3.0.<br>
    <a href="https://en.cppreference.com/w/cpp/error/overflow_error" class="_attribution-link">https://en.cppreference.com/w/cpp/error/overflow_error</a>
  </p>
</div>

				
			</div>
			<ins class="adsbygoogle"
     style="display:block"
     data-ad-client="ca-pub-2572770204602497"
     data-ad-slot="1992473792"
     data-ad-format="auto"
     data-full-width-responsive="true"></ins>
<script>
     (adsbygoogle = window.adsbygoogle || []).push({});
</script>
		</div>
	</section>

	</div>
	<svg style="display:none">
		<symbol id="icon-dir"><svg viewBox="0 0 20 20"><path d="M15 10c0 .3-.305.515-.305.515l-8.56 5.303c-.625.41-1.135.106-1.135-.67V4.853c0-.777.51-1.078 1.135-.67l8.56 5.305S15 9.702 15 10z"/></svg></symbol>
	  </svg>
</body>
</html>
